### PR TITLE
New design for ngo register

### LINKED ISSUE #11: New desing NGO registration
- must contain:
- [ ] new sidebar, only with text, delete all the other blocks from sidebar
- [ ] Registration form

[figma url](https://www.figma.com/proto/Fm1mdnskOPnJCX1AWgpR3U/Ajut.md_Web-Design_UI?scaling=min-zoom&node-id=106%3A1330)

image

### PR DESCRIPTION
**Pull request by vitaliel**
Closes #11

### CODE REVIEW

**Comment by vitaliel** [comment]
How it looks:
ngo-reg-form

File: ro_help/hub/templates/ngo/register_request.html [hunk 1]
```diff
@@ -23,6 +24,8 @@
                 </div>
                 </article>
             {% else %}
+                <h1>Înregistrare ONG</h1>
```
Comment (alexkingdom) [inline ro_help/hub/templates/ngo/register_request.html]: Seems we need to translate this string

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1145,13 +1070,13 @@ msgid ""
...
-"Ajut.md - это полнофункциональная цифровая платформа, которая может быть использована"
-"всеми организациями, которые активно участвуют в предотвращении распространения"
-"Covid-19. Основная цель палтформы - помочь организациям собрать"
-"ресурсы, которые им нужны. Платформа поможет вам найти больше всего"
-"насущные потребности, с которыми сталкиваются организации по всей стране. В результате, это"
-"облегчает вам выбор дела, которому вы жертвуете, или усилий, которые вы делаете"
-"способствовать."
+"Ajut.md - это полнофункциональная цифровая платформа, которая может быть "
+"использованавсеми организациями, которые активно участвуют в предотвращении "
+"распространенияCovid-19. Основная цель палтформы - помочь организациям "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1328,8 +1264,8 @@ msgid ""
 "The password reset link was invalid, possibly because it has already been "
 "used. Please request a new password reset."
 msgstr ""
-"Ссылка для сброса пароля недействительна, возможно, потому, что она уже была"
-"использована. Запросите новый сброс пароля."
+"Ссылка для сброса пароля недействительна, возможно, потому, что она уже "
+"былаиспользована. Запросите новый сброс пароля."
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1909,8 +1845,9 @@ msgid ""
 "For any questions or issues, as well as to ensure your rights in regards to "
 "the processing of personal data by Code for Romania, you can contact us at:"
 msgstr ""
-"По любым вопросам или проблемам, а также для обеспечения ваших прав в отношении"
-"обработки персональных данных Code for Moldova, вы можете связаться с нами по адресу:"
+"По любым вопросам или проблемам, а также для обеспечения ваших прав в "
+"отношенииобработки персональных данных Code for Moldova, вы можете связаться "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -1994,9 +1931,9 @@ msgid ""
 "                "
 msgstr ""
 "\n"
-"                    Платеж за %(ngo)s подтвержден! Спасибо"
-"за вашу поддержку в этот период! Любая помощь увеличивает наши возможности"
-"делать добро дальше. \n"
+"                    Платеж за %(ngo)s подтвержден! Спасибоза вашу поддержку "
+"в этот период! Любая помощь увеличивает наши возможностиделать добро "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

File: ro_help/locale/ru/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -2009,9 +1946,9 @@ msgid ""
 "                    "
 msgstr ""
 "\n"
-"                        Платеж за %(ngo)s обрабатывается."
-"Спасибо за вашу поддержку в это время! Любая помощь увеличивает нашу"
-"способность творить добро дальше.\n"
+"                        Платеж за %(ngo)s обрабатывается.Спасибо за вашу "
+"поддержку в это время! Любая помощь увеличивает нашуспособность творить "
```
Comment (alexkingdom) [inline ro_help/locale/ru/LC_MESSAGES/django.po]: Typo

**Review by alexkingdom — CHANGES_REQUESTED** [review]
As I saw you have two PRs with same changes or at least part of changes are same, I commented in both cases, but after we will migrate one of your PR, you will rebase so we will see what you made in PR.

**Review by alexkingdom — APPROVED** [review]

**Comment by alexkingdom** [comment]
PR is ok for me, waiting for @johnsmithm to see if also is ok, so we can merge it.
Thank you very much for contributing!

**Pull request merged**